webapp/locales/de.json

```diff
@@ -22,7 +22,9 @@
     "responsible": "Verantwortlicher gemäß § 55 Abs. 2 RStV ",
     "bank": "Bankverbindung",
     "germany": "Deutschland",
-    "code-of-conduct": "Verhaltenscodex"
+    "code-of-conduct": "Verhaltenscodex",
+    "termsAndConditionsCeckbox": "<a href=\"/terms-and-conditions\" target=\"_blank\">Nutzungsbedingungen</a> gelesen und verstanden",
+    "termsAndConditionsNoChecked": "Bitte lies und bestätige die Nutzungsbednigungen!"
```

Comment:
two typos:
`termsAndConditionsCeckbox` --> `termsAndConditionsCheckbox`
`termsAndConditionsNoChecked` --> `termsAndConditionsNotChecked`

or, maybe preferable because it gives translators a better idea of the expected sentence:
`termsAndConditionsRead` and `confirmTermsAndConditions`

Reply:
thx, I changed it after your suggestion.